src/main/java/org/opensearch/ad/cluster/ADClusterEventListener.java

```diff
@@ -96,27 +84,8 @@ public void clusterChanged(ClusterChangedEvent event) {
             }
             Delta delta = event.nodesDelta();
 
-            // Check whether it was a data node that was removed
-            boolean dataNodeRemoved = false;
-            for (DiscoveryNode removedNode : delta.removedNodes()) {
-                if (nodeFilter.isEligibleNode(removedNode)) {
-                    LOG.info(NODE_REMOVED_MSG + " {}", removedNode.getId());
-                    dataNodeRemoved = true;
-                    break;
-                }
-            }
-
-            // Check whether it was a data node that was added
-            boolean dataNodeAdded = false;
-            for (DiscoveryNode addedNode : delta.addedNodes()) {
-                if (nodeFilter.isEligibleNode(addedNode)) {
-                    LOG.info(NODE_ADDED_MSG + " {}", addedNode.getId());
-                    dataNodeAdded = true;
-                    break;
-                }
-            }
-
-            if (dataNodeAdded || dataNodeRemoved) {
+            if (delta.removed() || delta.added()) {
```

Comment:
is this log necessary? Wouldn't this already be logged in core during a cluster changed event like a node leaving or joining?

Reply:
I'd prefer to keep this log to make AD debug easier, in case any bug happen we can tell apart from core issue or our own issue easily. And this log won't print too much, just print when cluster starts generally.